src/app/page.tsx

```diff
@@ -1,11 +1,5 @@
-import { getServerSession } from "next-auth";
 import { redirect } from "next/navigation";
 
-import MajorSelectionPage from "./MajorSelectionPage";
-
 export default async function Page() {
-  const session = await getServerSession();
-  if (!session?.user.id) redirect("/planner");
-
-  return <MajorSelectionPage />;
+  redirect("/planner");
```

Comment:
Let's wait for Zack to decide what we want to do with the index page, but we shouldn't just redirect to another page.